### ISSUE TITLE
Getting "required group missing" for existing NXdata.

### ISSUE THREAD

**Issue by domna** [nexusformat/cnxvalidate#35]
I have a NXdata entry in one of my NeXus files, which nxvalidate complains about:

```
definition=NXmpes.nxdl.xml message="Required group missing" 
... nxdlPath=/NXentry/NXdata 
... sev=error dataPath=/entry dataFile=MoTe.mpes.nxs
```

It is properly written to the file in the path /entry/data with the following attributes:
```
@NX_class: NXdata
@axes: energy
@signal: intensity
energy_indices: 0
```
there are also two arrays of the same length /entry/data/energy and /entry/data/intensity holding the data values. I'm also able to plot the data in h5web without any problems. So as far as I see it the NXdata requirements are fully met and I don't understand why nxvalidate is complaining about the missing group. Am I missing something?

**Comment by mkoennecke** [comment]
Can I have the file you are trying to validate?

**Comment by domna** [comment]
Yes, the problem should be present in this file: https://box.hu-berlin.de/f/5cf128f60f58487cb282/
I think it contains the boolean error (#34 ), too.

**Comment by meghdadyazdi** [comment]
Hi, is it solved? https://github.com/nexusformat/cnxvalidate/issues/40
